todo_app/data/trello_items.py

```diff
@@ -0,0 +1,22 @@
+from flask import session, Flask
+import requests
+import dotenv 
+import os
+
+def get_items():
+    dotenv.load_dotenv()
```

Comment:
This line shouldn't be necessary if you are running your app with `poetry run flask run`